osu.Game.Rulesets.Taiko/Judgements/TaikoDrumRollJudgement.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Rulesets.Taiko.Judgements
+{
+    public class TaikoDrumRollJudgement : TaikoJudgement
+    {
+        public override bool AffectsCombo => false;
+
+        protected override int NumericResultFor(HitResult result) => 0;
+
+        protected override double HealthIncreaseFor(HitResult result) => 0;
```

Comment:
It's a bit harsh no? Not gaining any HP for completing a drumroll perfectly?

Reply:
But also not losing any?